Sources/MapboxMaps/Annotations/Generated/CircleAnnotationManager.swift

```diff
@@ -41,13 +41,17 @@ public class CircleAnnotationManager: AnnotationManager {
     /// Dependency required to add gesture recognizer to the MapView
     private weak var view: UIView?
 
-    internal init(id: String, style: Style, view: UIView, mapFeatureQueryable: MapFeatureQueryable, layerPosition: LayerPosition?) {
+    /// Indicates whether the style layer exists after style changes. Default value is `true`.
+    public private(set) var isPersistent: Bool
```

Comment:
Are we sure we want this to be optional behavior? In what situation would you want this to be false?

Reply:
fyi, we didn't expose this option to users on Android, we will always the persistent layer for annotations.